packages/react/collection/package.json

```diff
@@ -23,7 +23,8 @@
     "@radix-ui/react-slot": "workspace:*"
   },
   "peerDependencies": {
-    "react": "^16.8 || ^17.0 || ^18.0"
+    "react": "^16.8 || ^17.0 || ^18.0",
+    "react-dom": "^16.8 || ^17.0 || ^18.0"
```

Comment:
Why does this one need it?

Reply:
It consumes types from `@radix-ui/react-primitive`, does it still apply in that case?

Reply:
I guess it still does by the definition of peer deps sadly…